WORKSPACE

```diff
@@ -363,6 +363,32 @@ new_git_repository(
     shallow_since = "1690331196 +0000",
 )
 
+new_git_repository(
+    name = "perfetto",
+    commit = "d136f355bb32288122dd2a0273e4fab280654d3a",
+    remote = "https://android.googlesource.com/platform/external/perfetto.git"
+)
+
+# For use with perfetto
+new_git_repository(
+    name = "com_google_protobuf",
+    commit = "6a59a2ad1f61d9696092f79b6d74368b4d7970a3",
+    remote = "https://chromium.googlesource.com/external/github.com/google/protobuf"
+)
+
+# For use with perfetto
+new_local_repository(
```

Comment:
I think we'll have to define an actual file here – accessing the external link directly is not supported, I also tried to create a file alias in a number of ways but that doesn't seem to work